src/theme.js

```diff
@@ -95,27 +184,27 @@ export default (overrides = {}) => {
       backgroundColor: colors.blueLight,
       fontColor: colors.blueDark,
     },
-    gray: {
-      backgroundColor: colors.grayLight,
-      fontColor: colors.grayDark,
+    grey: {
+      backgroundColor: colors.greyLight,
+      fontColor: colors.greyDark,
     },
   };
 
   const alertVariants = badgeVariants;
 
   const heights = {
     xs: 28,
-    sm: 32,
-    md: 36,
+    sm: 24,
+    md: 32,
     lg: 40,
-    xl: 44,
+    xl: 48,
   };
 
   const fontSizes = {
     xs: 8,
     sm: 10,
```

Comment:
I think 12 makes more sense here.

```suggestion
    sm: 12,
```